docusaurus.config.js

```diff
@@ -218,6 +209,21 @@ module.exports = {
       },
     ],
   ],
+  themes: [
+    [
+      require.resolve("@easyops-cn/docusaurus-search-local"),
+      {
+        hashed: true,
+        indexBlog: false,
+        indexDocs: true,
+        docsRouteBasePath: "/",
+        ignoreFiles: [
+          /client.*/i,
+          /server.*/i,
```

Comment:
so this would ignore: https://docs.statsig.com/server/java/#bootstrap but let through https://docs.statsig.com/server/javaSdk#bootstrap ?

not sure I follow exactly how the regex here is working but sounds good

Reply:
No, that was the point in my pr summary.  "This might be over-removing files now - I think each sdk landing page doesnt exist now"

I think I've removed the actual sdk landing page from consideration now